InfiniteRecharge/src/main/java/frc/robot/subsystem/climber/ClimbSubsystem.java

```diff
@@ -12,10 +12,10 @@
         Extending, Retracting, Off;
     }
 
-    private boolean active = false;
+    private boolean active = false; //Sets the climber to be deactivated by default, even though booleans should be false by default

```

Comment:
Can you put these comments above the vars? It's easier to read on a smaller screen.